internal/util/validation/formats_test.go

```diff
@@ -88,3 +88,28 @@ func TestValidateGitRevision(t *testing.T) {
 		assert.NotEmpty(ValidateGitRevision(&val, "bad.image.ref"), fmt.Sprintf("value: %q", val))
 	}
 }
+
+func TestValidateSystemdUnitPattern(t *testing.T) {
+	assert := assert.New(t)
+
+	goodValues := []string{
+		"foo.service",
+		"service",
+		"foo[0-9].service",
+		"foo?.service",
+		"foo\\.service",
+	}
+	for _, val := range goodValues {
+		assert.Empty(ValidateSystemdName(&val, "good.unit"))
+	}
+
+	badValues := []string{
```

Comment:
simply because the valid length changed from 256 to 63 it might be useful to include a 64 character length string in this test